test/base/src/androidTest/kotlin/io/realm/test/shared/MutableRealmTests.kt

```diff
@@ -75,12 +75,12 @@ class MutableRealmTests {
 
     @Test
     fun cancelingWrite() {
-        assertEquals(0, realm.objects(Parent::class).count())
+        assertEquals(0, realm.query<Parent>().count().find())
```

Comment:
Again, this tests the aggregator and not just the size of the result. I feel that this is a bit overkill.